packages/amazonq/src/inlineChat/output/responseTransformer.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getLogger } from 'aws-core-vscode/shared'
+import { decode } from 'he'
+
+export function responseTransformer(response: string): string | undefined {
+    try {
+        return decode(response)
+    } catch (err) {
+        if (err instanceof Error) {
+            getLogger().error(err)
+        } else {
+            getLogger().error(`An unknown error occurred: ${err}`)
+        }
```

Comment:
please review https://github.com/aws/aws-toolkit-vscode/blob/master/docs/CODE_GUIDELINES.md#exceptions and https://github.com/aws/aws-toolkit-vscode/blob/master/docs/arch_develop.md#exceptions